data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/impl/ReceiverVerticle.java

```diff
@@ -121,7 +121,7 @@ public void handle(HttpServerRequest request) {
     final var requestContext = new RequestContext(request);
 
     // Look up for the ingress producer
-    IngressProducer producer = this.ingressProducerStore.resolve(request.path());
+    IngressProducer producer = this.ingressProducerStore.resolve(request.host(), request.path());
```

Comment:
Have you considered passing the request context directly?

Reply:
I am ok with that too, but this is easier to test (otherwise I need to build a request in the tests). Also, I don't need anything other than the host and the path here